test/e2e/e2e.go

```diff
@@ -2345,8 +2345,8 @@ var _ = ginkgo.Describe("host to host-networked pods traffic validation", func()
 				endPoints = append(endPoints, hostNetPod)
 				nodesHostnames.Insert(hostNetPod.Name)
 
-				// this is arbitrary and mutuated from k8s network e2e tests. We aim to hit all the endpoints at least once
-				maxTries = len(endPoints)*len(endPoints) + 30
+				// this is arbitrary but 10 maxTries should be more than enough; we are only hitting the local endpoint due to externalTrafficPolicy=local
+				maxTries = 10
```

Comment:
lgtm but few questions:

Hi,  what's the value today from len(endPoints)*len(endPoints) ? I see others points of the code doing the same logic, would make sense to make a generic code for it ?  

https://github.com/ovn-org/ovn-kubernetes/pull/2815/commits/b5c20c446dc7814d3cdd7bbebe243201d774003b#diff-65b73a6a62a4e5b8903b9f17f0e03a42365b127e7d86f006f9bec363b4be04b8R1819

Also, 10 seems a magic number, would be better have a constant?

Reply:
Hmm .. the k8s network e2e tests also just use a "heuristic" when they use  `len(endPoints)*len(endPoints) + 30`. In that case, they are trying from each host to each endpoint and add a random 30 retries to it.

Because we are only hitting the local endpoint, I decided to lower it to 10. I admit that its arbitrary, but not more than the rest of the heuristic that we (or the e2e framework) are using.

Ack for the constant.

Reply:
Actually, it can't be a constant. It sets the maxTries variable from a few lines earlier, around line 2305